src/pages/ShoppingCart.jsx

```diff
@@ -1,7 +1,38 @@
+import { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import Alert from 'react-bootstrap/Alert';
+import Button from 'react-bootstrap/Button';
+import { GlobalContext } from '../context/GlobalContext';
+import CartItem from '../components/CartItem';
 import { PageContainer } from '../components/styles/PageContainer.styled';
+import { SectionTitle } from '../components/styles/SectionTitle.styled';
 
 const ShoppingCart = () => {
-  return <PageContainer>shopping cart page</PageContainer>;
+  const { cart } = useContext(GlobalContext);
+  const cartTotal = cart.reduce(
+    (sum, prod) => sum + prod.price * prod.amount,
+    0
+  );
```

Comment:
Now that I'm thinking about, the total can be stored on the context to use it more easily